api/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+class UsersController < ApplicationController
+    before_action :authenticate_user!
+    def show
```

Comment:
I'm unclear on exactly why the show action has no code but this may just be from not knowing rails all that well. But even so, it might be nice to add a comment explaining why just for readability?